content/en/docs/instrumentation/js/resources.md

```diff
@@ -88,6 +87,9 @@ EnvDetector found resource. Resource {
 ...
 ```
 
+You can use the environment variable `OTEL_SERVICE_NAME` to set value of
+the `service.name` resource attribute.
+
```

Comment:
can you move this above the example and then update the example to also include OTEL_SERVICE_NAME, that way end users have a better understanding of what's going to happen

Reply:
@svrnm made the change, pls have a look and let me know what you think